sprint-planning/estimation/readme.md

```diff
@@ -1,8 +1,93 @@
+time allocation for the work to be complete.
+
+To determine sizing, the team will generally pick two or three stories that reflect
+a known amount of work of varying size. Usually, a simple, an average and a complex
+story will work best. Each story will be given a relative weight based on whether
+it is more like one of the "example" stories. Over time, the "example" stories may
+need to be revisited if the team feels that all of their stories are average or
+complex.
+
+### Planning Poker
+
+Possibly the most popular approach to sizing is called Planning Poker. This approach
+assigns sizing based on the fibonacci numbers 1,2,3,5,8,13,20,40. Teams will go so
+far as to purchase cards that reflect the numbers and distribute the cards during
+the planning session.
+
+The team would usually pick a simple story and assign it a value of 2. Then the
+team would pick another story that was more difficult and assign it a corresponding
+value...i.e. a 5 would be assigned to a story that is viewed to be more than twice
+as difficult as the 2 but not four times as much to make it an 8.
+
+### T-Shirt Sizes
+
+When using numbers, teams start to over analyze what the points should be or they
+try to begin to correlate points to hours on the clock. Teams start to pad their
+estimates so they can justify their work with their time sheet. When you see this
+start to happen, it may be time to remind the team this is about relative sizing
+not about hours to complete the work.
+
+One way to do this is to switch to T-Shirt sizing. With this approach you have extra
+small, small, medium, large, extra-large, and double extra-large. When you remove
+the correlation to numbers, the teams remember that their goal is only to determine
+how difficult the work is before committing to the work to be done. It also gives
+the product owner visibility that what was perceived as a simple task may be more
+costly and thus less desirable than a couple of other features of a smaller size.
+
+Another benefit of using a less granular approach is that the team won't spend so
+much time trying to come to agreement on the precise measurement of the work. Decisions
+on medium or large are easier than deciding a work item is 5 or 8 points. That said
+this will also put the agile team leader in the position of converting sizes into
+numeric values for the purpose of charting and tracking velocity over time.
+
+## Estimation in Planning
+
+As the continues the exercise of estimating in the planning meeting, it should be
+viewed as an exercise in relative sizing of the work. Comparing the expected work
+to work that has already been done is helpful for the team to understand how they
+view the complexity of the work.
+
+The team estimate how many points the team can deliver in the sprint and the team
+makes a commitment for the sprint. During the sprint, the team takes the highest
+priority items off of the backlog and works on them to completion. At the end of

```

Comment:
As at this moment the backlog is the one  prioritized and estimated do be complete in the sprint, I suggest to distinguish it as _sprint backlog_ to avoid any refence to the overall backlog of the project.